index.html

```diff
@@ -0,0 +1,393 @@
+                                        <p>The importance of grassroots initiatives for youth in agriculture. Small-scale entrepreneurs working in food in West Africa.</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-11">
+                                <div class="row card-contents">
+
+                                    <div class="col-3">
+                                        <img src="./images/images/program_icon_04.png" alt="">
+                                    </div>
+
+
+                                    <div class="col-4">
+                                        <h3>Workshop</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>We have time to share our thoughts and opinions with experts by topic.</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-11">
+                                <div class="row card-contents">
+
+                                    <div class="col-3">
+                                        <img src="./images/images/program_icon_05.png" alt="">
+
+                                    </div>
+
+                                    <div class="col-4">
+                                        <h3>cc party</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>Create opportunities to freely network with other young farmers in West Africa.</p>
+                                    </div>
+                                </div>
+                            </div>
+
+                        </div>
+                        <div class="row">
+                            <div class="col-1"></div>
+                            <div class="col-10 join_summit">
+                                <center>Join summit 2023</center>
+                            </div>
+
+                        </div>
+
+                    </div>
+
+
+                </div>
+                <div class="row down_part">
+                    <div class="col-12">
+
+                        <center>
+                            <span>
+                                <a href="##">See whole program </a>
+                            </span>
+                        </center>
+
+                    </div>
+                </div>
+            </section>
+
+            <!--Down section-->
+            <section class="container down_section" id="desktop_speakers">
+
+      <!--contents of speaker cards coming from DOM in javascript-->
+                
+
+            </section>
+            <!--Mobile speaker parts-->
+            <!--Down section-->
+            <section class="container down_section" id="mobile_skpeakers">
```

Comment:
- [ ] For your speaker card mobile version is different from the original design mobile version. Please try to follow the design guidelines in order to make it similar to the original design and make your website gives a good user experience. :+1: 

- [ ] Also make sure that your more button is similar to original design :+1: 
#### Original design
![Screenshot_20230108_004714](https://user-images.githubusercontent.com/6615394/211175653-15b5a505-4f48-40c7-89dd-fcb49dd02891.jpg)